app/assets/stylesheets/refills/_centered-navigation.scss

```diff
@@ -102,6 +104,7 @@
   }
 
   ul li.nav-link {
+    font-size: 1.2em;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered background, display, font-size, line-height, overflow, padding-right, text-align, width, z-index